webapp/channels/src/components/widgets/menu/menu.tsx

```diff
@@ -145,7 +134,14 @@ export default class Menu extends React.PureComponent<Props> {
                     id={listId}
                     ref={this.node}
                     style={styles}
-                    className={classNames('Menu__content dropdown-menu', this.props.className)}
+                    className={classNames(
+                        'Menu__content dropdown-menu',
+                        {
+                            openLeft,
```

Comment:
Unlike the other Menu widgets that needed `isMobile` for use in JS, this component used it in the CSS which can just use a media query to determine how `openLeft` and `openUp` should be styled